stable/zetcd/values.yaml

```diff
@@ -24,10 +24,12 @@ resources: {}
   #  cpu: 100m
   #  memory: 128Mi
 
+nodeSelector: {}
```

Comment:
This is not enough, you need to add this to deployment, check here for detail https://github.com/kubernetes/charts/blob/master/stable/aws-cluster-autoscaler/templates/deployment.yaml#L50-L53

Also please add some comments here, take here as reference https://github.com/kubernetes/charts/blob/master/stable/aws-cluster-autoscaler/values.yaml#L15-L18

Reply:
no need update the deployment as the deployment has already included the nodeSelector section:
https://github.com/kubernetes/charts/blob/aa0380f852fb8d71d57acc8f63306699adfb827f/stable/zetcd/templates/deployment.yaml#L42-L44

Reply:
I see, then it is better add some comments here as https://github.com/kubernetes/charts/blob/master/stable/aws-cluster-autoscaler/values.yaml#L15-L18

Reply:
already have comments.